verification/curator-service/api/test/index.test.ts

```diff
@@ -19,6 +20,10 @@ afterAll(() => {
     return mongoose.disconnect();
 });
 
+beforeEach(() => {
```

Comment:
I feel like this addition is the result of some sadness and pain and I want you to know that you're seen

Reply:
👀